src/resources/Incorporations/benefitCompany.ts

```diff
@@ -6,9 +6,8 @@ import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module
 export const BenefitCompanyResource: IncorporationResourceIF = {
   entityType: CorpTypeCd.BENEFIT_COMPANY,
   displayName: GetCorpFullDescription(CorpTypeCd.BENEFIT_COMPANY),
-  title: 'Benefit Company Statement',
-  description: 'This company is a benefit company and, as such, has purposes that include conducting its ' +
-    'business in a responsible and sustainable manner and promoting one or more public benefits.',
+  title: null,
```

Comment:
Why did you null this one but not the other ones?

Reply:
Because only this title is rendered but no any other entity type.

Reply:
How can the title be rendered when it is null?

Reply:
I mean the reason I null this title because it was rendered when it was "Benefit Company Statement". After I nullify this, it will not be rendered.

Reply:
So why did you nullify this title (for BEN) but not for CCC or ULC?

Reply:
Because statement template tag is only rendered if the entity is [here](BENhttps://github.com/bcgov/business-create-ui/blob/17a3e9c4c3d41b45d4f9bbe74e962026415bead7/src/views/Incorporation/IncorporationDefineCompany.vue#L3-L11). I need to nullify the title in the BEN resource file otherwise "Benefit Company Statement" will be displayed. Since that template tag is only rendered for BEN and no other entity type, I don't have to nullify their titles.

Reply:
I can nullify other titles for consistency purposes.

Reply:
Thanks for the code link. I see what you mean now.

I think it wasn't the original intent to display the company statement section for BENs only. I think it should have been displayed for BCs, CCs and ULCs also. However, that requirement has now changed.

Since that template section is no longer needed, I recommend deleting it, and the obsolete getters, and also delete the now-obsolete resource fields (title, description and statement).

Reply:
> I can nullify other titles for consistency purposes.

Thanks. It's not just consistency -- they're obsolete now.

It's good to delete obsolete stuff to avoid the "Decoy Pattern" -- this is kind of a joke name but it's a real thing that some things can look like they're doing something but actually they're not.

Reply:
Thank you so much for your feedback. I felt very hesitant to delete this component, knowing that it will become obsolete in this ticket. I thought that this component might be used again later, but I will go on to remove this component in my next commit. The subsequent fields in the resources will be deleted, as well as the two getters for the title and the description.

Reply:
I also hesitate to just delete all of this, but we can always go back in git history and find this code and restore it later, if needed.

Reply:
Absolutely. Good point.

Reply:
Looks good.